math-magicians-app/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
- [ ] I kindly suggest that you update your [README](https://github.com/microverseinc/readme-template) in your repo, using the Microverse [template](https://github.com/microverseinc/readme-template) as this will give users detailed information about your project.
- [ ] Please remember to personalize your README file content and put your project information there.